### PR TITLE
Billing Models + Testing Coverage ... take 3

### PR DESCRIPTION
**Pull request by barbaraliau**
Cleaner pull request.

Includes updates for billing and test coverage. Updates include changes mentioned in previous pull requests.

### CODE REVIEW

File: lib/models/mirror.js [hunk 1]
```diff
@@ -15,9 +16,13 @@ var Mirror = new mongoose.Schema({
   },
   contact: {
     type: String,
-    ref: 'Contact'
+    ref: 'Contact',
+    required: true
```
Comment (braydonf) [inline lib/models/mirror.js:19]: I've noticed that there are existing records without a contact for mirrors, which has been a mystery....

**Comment by braydonf** [comment]
To save myself from making many comments again, I'll summarize:
- lodash not necessary in mirror, and not included in package.json. It's likely being required by a dependency and with npm@3 is at the highest level and able to be required (by chance). So we can add it in to package.json, or change to not need it.
- There are many stray console.logs in the adapters.

File: package.json [hunk 2]
```diff
@@ -27,21 +27,25 @@
   },
   "homepage": "https://github.com/Storj/service-storage-models#readme",
   "dependencies": {
+    "dotenv": "^2.0.0",
     "elliptic": "^6.3.2",
     "hat": "0.0.3",
     "merge": "^1.2.0",
     "mime-db": "^1.24.0",
+    "mnemonic": "^1.0.1",
```
Comment (braydonf) [inline package.json]: Accidentally added?

File: lib/models/mirror.js (hunk 1 above)
Comment (barbaraliau) [reply lib/models/mirror.js:19]: Are contacts required?

File: package.json (hunk 2 above)
Comment (barbaraliau) [reply package.json]: Removed.

File: lib/models/mirror.js (hunk 1 above)
Comment (braydonf) [reply lib/models/mirror.js:19]: Yeah, otherwise they get filtered out: https://github.com/Storj/bridge/blob/master/lib/server/routes/reports.js#L117

Comment (braydonf) [reply lib/models/mirror.js:19]: I can test to make sure that this won't cause issues with existing data.

**Comment by barbaraliau** [comment]
@braydonf I think I got all the stray console.logs and updated mirror.js. Let me know if there are other things that need to be changed.

**Comment by braydonf** [comment]
Okay, looking over again. And can the start to do some manual testing.

**Comment by barbaraliau** [comment]
Thanks 👍

File: lib/models/mirror.js (hunk 1 above)
Comment (braydonf) [reply lib/models/mirror.js:19]: Okay, manually tested, and there are not any issues. Any mirror that doesn't have a contact, *for some reason*, will yield an error such as this, on the cli side that doesn't handle the situation: https://github.com/Storj/core-cli/issues/82

**Comment by braydonf** [comment]
Okay I went through and tested, register, upload/download, mirroring with https://github.com/braydonf/storj-integration/tree/billing-models-updates and everything is working. I haven't tested any of the billing model changes yet, or some of the changes to user.

**Comment by braydonf** [comment]
Okay found an issue, I'm getting these in bridge logs:
```
Error: ENOENT: no such file or directory, open '.env'
```
From: storj-service-storage-models/index.js:6:19

**Comment by barbaraliau** [comment]
@dylanlott @bryanchriswhite I believe this warning is from requiring the `dotenv` package. What is your preferred way to handle this? I can suppress the warning with `require('dotenv').config({silent: true});`

**Comment by braydonf** [comment]
Also I noticed that `queries/*` exist in both the billing repo and this PR. In general, I think it's best to have billing related logic in the billing repository, unless there is a reason for it existing here.

**Comment by barbaraliau** [comment]
Ah. Good catch. It wasn't even being used, so I removed it.

File: lib/models/user.js [hunk 3]
```diff
@@ -6,6 +6,11 @@ const activator = require('hat').rack(256);
 const crypto = require('crypto');
 const mongoose = require('mongoose');
 const SchemaOptions = require('../options');
+const constants = require('../constants');
+const DEBIT_TYPES = constants.DEBIT_TYPES;
+const CENTS_PER_GB_HOUR = constants.CENTS_PER_GB_HOUR;
```
Comment (braydonf) [inline lib/models/user.js]: Are these constants being used here? I don't think they are here anymore.

Comment (barbaraliau) [reply lib/models/user.js]: Nope. All gone.

**Comment by braydonf** [comment]
Okay, so this looks good to me to merge. We can iterate with improvements, fixes and increased test coverage.

**Comment by braydonf** [comment]
Let's let this soak on master for a bit and then, we can bump, tag, release.

**Comment by barbaraliau** [comment]
Sounds good to me.

**Pull request merged**